### ISSUE TITLE
Nuisance `reportMissingImport` warning when explicitly handled optional module is missing

### ISSUE THREAD

**Issue by telamonian** [microsoft/pylance-release#1005]
Not sure if this is a pyright bug or an issue with how pylance configures pyright. For now, I'd settle for a vscode -> pylance setting that turns off all `reportMissingImport` warnings.

## Environment data

-   Language Server version: 2021.2.4
-   OS and version: OSX 10.14.6
-   Python version (& distribution if applicable, e.g. Anaconda): py3.7

## Expected behaviour

The static checking of the ide should not raise a warning or error for a missing optional python module:

```python
try:
    import hdf5plugin
except ImportError:
    pass
```

In other words, because my code is explicitly handling the case when `hdf5plugin` is missing/unimportable, the above should be treated as normal, valid code.

## Actual behaviour

This block raises a nuisance `reportMissingImport` warning if `hdf5plugin` is not installed:

```python
try:
    import hdf5plugin
except ImportError:
    pass
```

**Comment by jakebailey** [comment]
Duplicate of #32; thumbs up that issue if you'd like to see that behavior changed.

**Comment by jakebailey** [comment]
Note that you can silence this by adding `# type: ignore` to the line.

**Comment by telamonian** [comment]
> Duplicate of #32; thumbs up that issue if you'd like to see that behavior changed.

Thanks for the pointer! I did that

**Comment by telamonian** [comment]
> Note that you can silence this by adding `# type: ignore` to the line.

Also, thanks for this. That will at least fix my immediate problem
